src/Popup.tsx

```diff
@@ -5,7 +5,7 @@ import { IPopupPickerProps } from 'rmc-picker/lib/PopupPickerTypes';
 
 export interface IPopupDatePickerProps extends IPopupPickerProps {
   datePicker: React.ReactElement<IDatePickerProps>;
-  onChange?: (date?) => void;
+  onChange?: (date?: any) => void;
```

Comment:
should there be a Date Object?

Reply:
not sure I just need a quick fix so we can merge `refactor-ts`

Reply:
pretty sure it should be Date.